src/bootstrap/bootstrap.py

```diff
@@ -891,7 +891,11 @@ def bootstrap(help_triggered):
     build.clean = args.clean
 
     try:
-        with open(args.config or 'config.toml') as config:
+        toml_path = args.config or 'config.toml'
+        if not os.path.exists(toml_path):
```

Comment:
Do we even need this check?

Reply:
Yes, or it will break the current behavior where you can have per-directory configuration. This is the same way git handles `.gitignore`s.